plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/m2e/core/ui/preferences/MavenErrorPreferencePage.java

```diff
@@ -0,0 +1,130 @@
+ * @author jkopriva, ldimaggi
+ *
+ */
+
+public class MavenErrorPreferencePage extends PreferencePage {
+
+	private static final String GROUP_ID = "\"groupId\" duplicate of parent groupId";
+	private static final String VERSION = "\"version\" duplicate of parent version";
+	private static final String PROJECT_CONFIG = "Out-of-date project configuration";
+	private static final String PLUGIN_EXECUTION = "Plugin execution not covered by lifecycle configuration";
+	private static final String MANAGED_VERSION = "Overriding managed version";
+	private static final Logger log = Logger.getLogger(MavenErrorPreferencePage.class);
+
+	/**
+	 * Construct the preference page with "Maven" > "Errors/Warnings".
+	 */
+	public MavenErrorPreferencePage() {
+		super(new String[] { "Maven", "Errors/Warnings" });
+	}
+	
+	/**
+	 * Getter method for "groupId duplicate of parent groupId" LabeledCombo
+	 * @return String value of selected item in labeled combo
+	 */	
+	public String getGroupId () {
+		return new LabeledCombo(GROUP_ID).getSelection();
+	}
+	
+	/**
+	 * Setter method for "groupId duplicate of parent groupId" LabeledCombo
+	 * @param groupId
+	 */
+	public void setGroupId (String groupId) {
+		new LabeledCombo(GROUP_ID).setSelection(groupId);
+		this.apply();
+		log.info("Set Maven error/warning Group ID to: " + groupId);
+	}
+		
+	/**
+	 * Getter method for "version duplicate of parent version" LabeledCombo
+	 * @return String value of selected item in labeled combo
+	 */	
+	public String getVersion () {
+		return new LabeledCombo(VERSION).getSelection();
+	}
+	
+	/**
+	 * Setter method for "version duplicate of parent version" LabeledCombo
+	 * @param version  
+	 */
+	public void setVersion (String version) {
+		new LabeledCombo(VERSION).setSelection(version);
+		this.apply();
+		log.info("Set Maven error/warning Version to: " + version);
+	}
+	
+	/**
+	 * Getter method for "Out-of-date project configuration" LabeledCombo
+	 * @return String value of selected item in labeled combo
+	 */		
+	public String getProjectConfig () {
+		return new LabeledCombo(PROJECT_CONFIG).getSelection();
+	}
+	
+	/**
+	 * Setter method for "Out-of-date project configuration" LabeledCombo
+	 * @param version  
+	 */
+	public void setProjectConfig (String projectConfig) {
+		new LabeledCombo(PROJECT_CONFIG).setSelection(projectConfig);
+		this.apply();
+		log.info("Set Maven error/warning Project Config to: " + projectConfig);
+	}
+	
+	/**
+	 * Getter method for "Plugin execution not covered by lifecycle configuration" LabeledCombo
+	 * @return String value of selected item in labeled combo
+	 */		
+	public String getPluginExecution () {
+		return new LabeledCombo(PLUGIN_EXECUTION).getSelection();
+	}
+	
+	/**
+	 * Setter method for "Plugin execution not covered by lifecycle configuration" LabeledCombo
+	 * @param version  
+	 */
+	public void setPluginExecution (String pluginExecution) {
+		new LabeledCombo(PLUGIN_EXECUTION).setSelection(pluginExecution);
+		this.apply();
```

Comment:
Remove this apply. Apply should be called on a preference page separately by a user.